config/migrations/2024/20240212141020-onboarding-public-ocmw-associations/202402141191-import-public-ocmw-association-contact-details.sparql

```diff
@@ -0,0 +1,1884 @@
+
+        INSERT {
+          GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+            ?contact <http://schema.org/telephone> "tel:+3234508400" .
+            ?contact <http://xmlns.com/foaf/0.1/page> "https://www.woonzorgnetwerkedegem.be/" .
+            ?contact <http://schema.org/email> "immaculata@wznedegem.be" .
+          }
+        }
+        WHERE {
+          GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+            ?adminUnit <http://www.w3.org/ns/adms#identifier> ?id ;
+              <http://www.w3.org/ns/org#hasPrimarySite> ?primarySite .
+            ?id <http://www.w3.org/2004/02/skos/core#notation> "KBO nummer" ;
+              <https://data.vlaanderen.be/ns/generiek#gestructureerdeIdentificator>/<https://data.vlaanderen.be/ns/generiek#lokaleIdentificator> "0685516024" .
+
+            ?primarySite <http://www.w3.org/ns/org#siteAddress> ?contact .
+
+            ?contact <http://schema.org/contactType> "Primary" .
+
+            OPTIONAL { ?contact <http://schema.org/telephone> ?phone . }
+            OPTIONAL { ?contact <http://xmlns.com/foaf/0.1/page> ?website . }
+            OPTIONAL { ?contact <http://schema.org/email> ?email . }
```

Comment:
They do not have a goal anymore. The version of the generation script I started from also had to remove any contact details that were added by users in between the initial onboarding and the addition of the contact details sometime later. I removed that deletion part since it is not relevant to this onboarding, but missed these lines apparently.